via/static/scripts/video_player/hooks/use-app-layout.ts

```diff
@@ -20,6 +20,7 @@ export function useAppLayout(
 
   const updateWidth = useCallback(() => {
     const containerWidth = appContainer.current?.clientWidth;
+    /* istanbul ignore next */
```

Comment:
Mocking the hook in `VideoPlayerApp`'s tests kicked up this test miss. Happy to do something more elegant if desired.

Reply:
I think this is fine.